internal/pkg/operand/strategy.go

```diff
@@ -0,0 +1,24 @@
+package operand
+
+// DeletionStrategy describes how to handle operands on-cluster when deleting the associated operator.
+type DeletionStrategy string
+
+const (
+	// Cancel is the default deletion strategy: it will cancel the deletion operation if operands are on-cluster.
+	Cancel DeletionStrategy = "cancel"
+	// Ignore will ignore the operands when deleting the operator, in effect orphaning them.
+	Ignore DeletionStrategy = "ignore"
+	// Delete will delete the operands associated with the operator before deleting the operator, allowing finalizers to run.
+	Delete DeletionStrategy = "delete"
+)
+
+func (d DeletionStrategy) String() string {
+	return string(d)
+}
+
```

Comment:
Since `DeletionStrategy` is already a string, this function seems unnecessary. If we ever need to have a custom `String()` function, I think we can add it later.